internal/push/push.go

```diff
@@ -34,8 +39,15 @@ func pushRepo(repoName string) {
 
 	logger.Waitingf("Initialising Git project %s...", repoName)
 
-	// we have to do a `plainOpen` so the git library can perform actions on it that we need below.
-	r, err := git.PlainInit(configure.CreateRepositoryDirectory+repoName, false)
+
+    var wt, dot billy.Filesystem
```

Comment:
Might need to reindent this using tabs instead of spaces. I know go can be weird with indentations. `go fmt` should do the trick! Happy to merge after that